benchmarks/vector_db_compare/test_system_qualification.py

```
@@ -0,0 +1,253 @@
+                                    "result_count": 10000,
+                                    "errors": 0,
+                                    "timeouts": 0,
+                                    "recall_at_10": 0.95,
+                                    "qps": 1.0,
+                                    "p50_nanos": 1,
+                                    "p95_nanos": 2,
+                                    "p99_nanos": 3,
+                                },
+                            }
+                            if contract["system"] == "treedb":
+                                cell["counters"] = {key: 0 for key in self.plan["artifact_contract"]["tree_db_counters"]}
```

Ruff (0.16.1)

[warning] Unnecessary dict comprehension for iterable; use `dict.fromkeys` instead

Replace with `dict.fromkeys(iterable)`)

(C420)